blog/2021-09-07-introducing-capact.md

```diff
@@ -12,16 +12,15 @@ tags:
 ---
 
-Managing the lifecycle of infrastructure, applications, and processes in modern IT is problematic.
-There are a variety of tools and best practices, which are ever-changing. It's a struggle to stay up-to-date with these practices, and not everyone is an expert at everything. Technical debt is inevitable.
+Managing the lifecycle of infrastructure, applications, and processes in modern IT is problematic. There are a variety of tools and best practices, which are ever-changing. It's a struggle to stay up-to-date with these practices, and not everyone is an expert at everything. Technical debt is inevitable.
 
-We are ultimately all alone. Siloed within the context of team, department, or company, locked-in in a particular ecosystem of tooling. To deliver end-to-end capabilities, we build them from scratch in a vacuum. We consume API calls, transform data, build and manage infrastructure and applications. Do we all really have different use cases? Don't we spend too much time solving the same problems as others have?
+We are ultimately all alone. We are siloed within the context of team, department, or company, locked-in in a particular ecosystem of tooling. To deliver end-to-end capabilities, we build them from scratch in a vacuum. We consume API calls, transform data, build and manage infrastructure and applications. Don't we spend too much time solving the same problems as others have?
 
-What if we could share the expertise and, as a result, save time? Collaborating together, we ensure we use the best tools out there, and follow the best practices. This is something that already happens—we have plenty of libraries and frameworks out there. But what if we went a step further and have a way to create, use and share building blocks that are language-agnostic abstracted capabilities?
+Sharing expertise is something that already happens—we have plenty of libraries and frameworks out there. But what if we went a step further and have a way to create, use and share building blocks that are **language-agnostic abstracted capabilities**?
 
-For example, if you're not a cloud expert, all you need to know is that you want a managed PostgreSQL database on AWS and let others make it. In that way, services are managed by subject-matter experts, and you can simply focus on your business logic. Likewise, if you need any Kubernetes cluster—by saying "I want any Kubernetes cluster" and letting the magic happen, you could cover both local development and production scenarios...
+For example, if you're not a cloud expert, all you need to know is that you want a managed PostgreSQL database on AWS. Let the subject-matter experts take care of it, and simply focus on your business logic. Likewise, if you need any Kubernetes cluster—by saying "I want any Kubernetes cluster" and letting the magic happen, you could cover both local development and production scenarios…
 
-What if I told you... **it is all possible**?
+What if I told you… **it is all possible**?
```

Comment:
it is all possible --> Capact makes it possible.

Reply:
There's the following sentence right after "What if I told you… it is all possible?":

> Introducing Capact, crowdsourced expertise. Learn once, use everywhere.

🤔 Personally I like the current version more, as at this point the reader doesn't know what's Capact.

Maybe we could keep it as it is, or alternatively have something like:

"What if I told you… there's a project that makes it possible?"

Reply:
Good point. Let's go with: "now this is all possible."

Reply:
```suggestion
What if I told you… **now this is all possible**?
```